snooty/main.py

```diff
@@ -89,6 +100,21 @@ class MongoBackend(Backend):
     def __init__(self, connection: pymongo.MongoClient) -> None:
         super(MongoBackend, self).__init__()
         self.client = connection
+        self.db = self._config_db()
+
+    def _config_db(self) -> str:
+        with PACKAGE_ROOT.joinpath("config.toml").open(encoding="utf-8") as f:
+            config = toml.load(f)
+            db_name = config["environments"][SNOOTY_ENV]["db"]
+            assert isinstance(db_name, str)
+            return db_name
+
+    def _manage_indexes(self) -> None:
+        # List of indexes to be created. For now, metadata and documents collections use the same indexes.
+        indexes = [IndexModel("page_id", unique=True)]
+
+        self.client[self.db][COLL_DOCUMENTS].create_indexes(indexes)
+        self.client[self.db][COLL_METADATA].create_indexes(indexes)
```

Comment:
I'm not clear on the benefits of creating an index here? I'm imagining we will set this at db setup.

Reply:
Yes true, this can be removed!